### PR TITLE
Fix running bash script on Windows

### PR DESCRIPTION
**Pull request by CyanDevs**
Signed-off-by: Chris Yan <chrisyan@microsoft.com>

### CODE REVIEW

**Review by RRathna — APPROVED** [review]

**Review by RRathna — COMMENTED** [review]
@CyanDevs does `touch "$lock"` work in windows?

**Comment by CyanDevs** [comment]
> @CyanDevs does `touch "$lock"` work in windows?

If I remember correctly, on msys and cygwin they are installed by default. Here's a test on a Windows container with git bash installed (msys):
image

**Review by radhikaj — APPROVED** [review]

**Comment by radhikaj** [comment]
Bors r+

File: tools/oeutil/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ if (BUILD_ENCLAVES)
     COMMAND openssl genrsa -out oeutil_enc_private.pem -3 3072
     COMMAND openssl rsa -in oeutil_enc_private.pem -pubout -out
             oeutil_enc_public.pem
-    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/gen_pubkey_header.sh oeutil_enc_pubkey.h
+    COMMAND bash ${CMAKE_CURRENT_SOURCE_DIR}/gen_pubkey_header.sh oeutil_enc_pubkey.h
```
Comment (anakrish) [inline tools/oeutil/CMakeLists.txt:13]: I think elsewhere in cmake we use GIT_BASH

Comment (CyanDevs) [reply tools/oeutil/CMakeLists.txt:13]: Thanks!

**Review by anakrish — APPROVED** [review]

**Comment by CyanDevs** [comment]
After using OE_BASH, CodeQL check is failing. Need to check why it is happening.

**Comment by CyanDevs** [comment]
The command for `gen_pubkey_header.sh` + parameters needed to be wrapped in quotes but was also too long to fit in a line, so the current proposed line is used to avoid the overly lengthy line. It results in a peculiar looking command that looks like so, but it still works:
```
[  9%] Generating oeutil_enc_private.pem, oeutil_enc_public.pem, oeutil_enc_pubkey.h
cd /openenclave/build/tools/oeutil && openssl genrsa -out oeutil_enc_private.pem -3 3072
Generating RSA private key, 3072 bit long modulus (2 primes)
....................................++++*...................................................................................++++
...............++++*........................................................................................................................................................................................................................................................++++*.....................................................................................................................................................................................++++*...........................................................................................................................................++++
e is 3 (0x03)
cd /openenclave/build/tools/oeutil && openssl rsa -in oeutil_enc_private.pem -pubout -out oeutil_enc_public.pem
writing RSA key
cd /openenclave/build/tools/oeutil && /usr/bin/bash -c /openenclave/tools/oeutil/gen_pubkey_header.sh\ oeutil_enc_pubkey.h\ oeutil_enc_public.pem
oeutil_enc_pubkey.h
oeutil_enc_public.pem
make[2]: Leaving directory '/openenclave/build'
```

**Pull request merged**